Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM docker.io/library/golang:1.21@sha256:0b27e9d0b6058d169f02523a20497f89322aa0726ee678dc414a92c6ca8f52df as builder
+FROM docker.io/library/golang:1.21@sha256:62e5883c278354041360e26dd3ba8c62c8621743a2b4fd7f0f0caf00af45d15c as builder
```

Comment:
The tag `docker.io/library/golang:1.21.1` also exists now, so we could use that to increase transparency, instead of just bumping digests

Reply:
Fixed in https://github.com/statnett/image-scanner-operator/pull/562